_posts/2016-04-05-nectar-overview.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+cover: 'assets/images/general-cover-3.jpg'
+title: NectarCommerce Overview
+tags: docs
+subclass: 'post tag-docs'
+categories: 'elixir'
+author: 'Pikender'
+navigation: true
+logo: 'assets/images/nectar-cart.png'
+---
+
+Vision
+======
+
+**E-commerce Framework with an extension methodology which allows easy integration of custom features unique to the domain.**
+
+## NectarCommerce in NutShell
+
+[NectarCommerce](https://github.com/vinsol/nectarcommerce) is an Elixir/Phoenix Application which is currently having following E-commerce features:
+
+- Product Management
+- Order Management
+- Basic Stock Management
+
+Please check out the [demo](https://github.com/vinsol/nectarcommerce#demo) and [README](https://github.com/vinsol/nectarcommerce/blob/master/README.md)
+
+## NectarCommerce and Extensions
+
+Project is in its early stages where NectarCommerce and extensions are residing side-by-side in _Elixir umbrella app_.
+
+Plan is to evolve where Nectar and all other extensions will be phoenix projects, which can be downloaded as hex packages.
```

Comment:
Some extensions may not be a phoenix project(like tax calculators or mailing services), rather we can say mix applications which may have web components via phoenix or something like that.